### PR TITLE
Revamps "Station AI" announcer

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
This PR updates the sound files for the old "Station AI" that are used for things such as:
Shuttle being called, recalled, docking. Blob and virus biohazard alert. Round-start cue. And so on.

- All old sounds are updated
- New lines are added for changes in security levels
- New lines are added for some minor, moderate, and major events
- Crew transfer and emergency shuttle announcers are now different
- Biohazards all have a voice line.

I've also included a new folder under "tools" that will aid in the generation of more lines in future, so we don't just repeat the same issue with losing our method of making these.

Here are some previews of the new voice, they have to be in mp4 due to GitHub limitations.

https://user-images.githubusercontent.com/12197162/116815548-f66f6f00-ab55-11eb-9540-fc42d7b7f012.mp4

https://user-images.githubusercontent.com/12197162/116815549-f66f6f00-ab55-11eb-8b15-0926327a78ba.mp4

https://user-images.githubusercontent.com/12197162/116815550-f7080580-ab55-11eb-834a-196f9f24d77e.mp4

https://user-images.githubusercontent.com/12197162/116815551-f7080580-ab55-11eb-9058-5e86b7cdeb66.mp4

https://user-images.githubusercontent.com/12197162/116815552-f7a09c00-ab55-11eb-8102-2f470e60d777.mp4

https://user-images.githubusercontent.com/12197162/116815553-f7a09c00-ab55-11eb-9d8c-bd8eabf482ef.mp4

## Why It's Good For The Game
Alright, this is a bit touchy. We all hold these soundbites dear to our hearts, and they evoke strong nostalgic feelings.
Unfortunantely, the method to generate them is utterly lost to the ages. Trust me, I did a LOT of digging.
So, in order to bring new announcer lines and tweak old ones, the system needs to be redone.
I have generated a new voice, with the help of @stylemistake on discord, that we know the parameters for, and can thus customise in the future.
We now get more unique event announcer lines, as well as separate announcers for crew transfer and emergency shuttles.

## Changelog
:cl:
soundadd: Replaced old station AI lines, added new ones.
sounddel: Removed unused station AI lines.
/:cl:

### CODE REVIEW

**Comment by Wojz** [comment]
I feel a good amount of people will oppose this for nostalgia reasons but keeping this old system means we can't add new announcements without them being jarring. There's no other way.

File: tools/announcement_voice_filter/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#AI Announcement Voice Effects Template#
+This template file helps apply the correct sound effects to generate further Station AI announcement messages.
```
Comment (AffectedArc07) [inline tools/announcement_voice_filter/README.md:2]: Make this a valid MD file. It looks bad.
image

Comment (S34NW) [reply tools/announcement_voice_filter/README.md:2]: oof, it does look stinky. I'll fix it up

**Comment by SabreML** [comment]
I'm fine with this tbh, although I do think the current ones sound better even discounting the nostalgia factor.
The one issue I have is that they all sound pretty fast, almost sped up compared to the current stuff.

File: tools/announcement_voice_filter/README.md (hunk 1 above)
Comment (S34NW) [reply tools/announcement_voice_filter/README.md:2]: image
done

**Comment by S34NW** [comment]
I could slow them down a smidge and see if it improves things

**Comment by stylemistake** [comment]
I think you could've used a better, nicer sounding TTS engine. I've used Amazon Polly for my experiments. She sounds hot, as she should.

**Comment by S34NW** [comment]
This is amazon polly :)

**Comment by stylemistake** [comment]
Well I remember there was a wide selection of voices for en_US.

**Comment by ManiacalSymphony** [comment]
I always found it odd that the automated announcement and the station AI sounded different, maybe hopefully fingers crossed the AI itself gets refurbished

**Comment by Anticept** [comment]
https://github.com/N3X15/ss13-vox

**Comment by S34NW** [comment]
> 
> 
> https://github.com/N3X15/ss13-vox

That is for player AI announcements, NOT the generic station announcer AI voice. They are two entirely separate things.

**Comment by Anticept** [comment]
Fox McCloud was the one who committed the original announcement soundbytes. I was told several times what was used to generate the voice, let me see if I can still reach the people who knew.

**Comment by Anticept** [comment]
I only got an answer back from one person, and they can't remember the name of the program either. It was a commercially available TTS engine and I believe the output was further transformed with reverb and some high pass filtering. Sorry :(

Good luck!

**Comment by S34NW** [comment]
Thanks, I did similar digging and came up blank.
The very difficulty you mentioned is what prompted me to do this, it's time to streamline.

**Comment by Spacemanspark** [comment]
Bro these new sounds are dope, reminds me of something from Tron

I'd gladly dump our current sounds for this

**Review by Kyep — APPROVED** [review]

**Review by variableundefined — APPROVED** [review]

**Comment by TerraPlanet** [comment]
I mean, the old ai was using Half Lifes Vox announcer.

**Comment by Anticept** [comment]
I don't remember any half life vox that sounded like the old AI

**Comment by Spacemanspark** [comment]
Paradise used the half life vox announcer once upon a time, but went to the female voice announcer around 2015/2016.

**Pull request merged**